code/hash/Hacl.Hash.Core.SHA2.fst

```diff
@@ -60,7 +60,8 @@ let index_h (a: sha2_alg) (i: U32.t): ST.Stack (word a)
     | SHA2_384 -> B.recall h384; IB.recall_contents h384 Constants.h384; h384.(i)
     | SHA2_512 -> B.recall h512; IB.recall_contents h512 Constants.h512; h512.(i)
 
-open Hacl.Hash.Core.SHA2.Constants
+let k224_256 = IB.igcmalloc_of_list HS.root Constants.k224_256_l
+let k384_512 = IB.igcmalloc_of_list HS.root Constants.k384_512_l
```

Comment:
why this change? as far as I remember it was important to have these constants exposed as a separate module per the comment regarding Vale... maybe it was to avoid vale taking a dependency on the whole code?

Reply:
This I don't know, it was in the `protz_wasm` branch when I took it over